### DISCUSSION TITLE
Issues with Contenful Example

### DISCUSSION THREAD

**Discussion by doodirock** [vercel/next.js#27810] [Help]
I've recently started to test out Contenful for Vercel and have been using https://github.com/vercel/next.js/tree/canary/examples/cms-contentful for my example. I'm not sure if its up to date, but no matter what I do I'm getting errors on validation with a message stating:

`NotFound: Status: 404 - Not Found - Message: The resource could not be found. - Request ID:`

![help](https://user-images.githubusercontent.com/3209701/128463707-1445a06b-60a5-478d-83d3-76553bd7a9b9.png)

I've checked about 50 times to see if my space ID is correct.  I've deleted and recreated spaces 5 times.  I've manually added content models and tried the script. No matter what I do I get the same error as above.  Env var are set up correctly as well.  I'm at a loss.  Is this a contentful issue?

Node 14+

**Comment by doodirock** [comment]
It appears as though the issue has something to do with windows vs Mac. As a guess I went ahead and tried everything on my Mac and was able to validate the content models. 

However a new issue arose. Even though content exists, if I run build, dev, or preview I get only a blank page. It doesn't seem to actually pull any content from contentful. There are no errors and no way to really see what is being pulled.

**Reply by wlaj** [reply]
I'm experiencing the same thing. The starter seems to work fine for sanity. I asked a similar question on the contentful slack. Did you find a work-around or ended up using another service?

**Reply by doodirock** [reply]
No I just move away from contentful

**Reply by ItsJamesMurray** [reply]
any update here?  The butterCMS one caused an error right off the bat so this was my second CMS example.

**Reply by wlaj** [reply]
The contentful team said they were working on fixing it, but no updates yet, it seems. I personally am using GraphCMS, it's among the list of examples ( and works ). it's got a soft API call limit of 1 million, compared to contentful's 2 million but it supports multiple users and it's interface is easier to use, in my opinion.
